include/deal.II/base/derivative_form.h

```diff
@@ -89,46 +91,48 @@ class DerivativeForm
   operator=(const Tensor<1, dim, Number> &);
 
   /**
-   * Converts a DerivativeForm <order,dim,dim> to Tensor<order+1,dim,Number>.
-   * In particular, if order==1 and the derivative is the Jacobian of F, then
-   * Tensor[i] = grad(F^i).
+   * Converts a DerivativeForm <order, dim, dim, Number> to Tensor<order+1, dim,
+   * Number>. In particular, if order == 1 and the derivative is the Jacobian of
+   * $\mathbf F(\mathbf x)$, then Tensor[i] = $\nabla F_i(\mathbf x)$.
    */
   operator Tensor<order + 1, dim, Number>() const;
 
   /**
-   * Converts a DerivativeForm <1, dim, 1> to Tensor<1,dim,Number>.
+   * Converts a DerivativeForm<1, dim, 1, Number> to Tensor<1, dim, Number>.
    */
   operator Tensor<1, dim, Number>() const;
 
   /**
-   * Return the transpose of a rectangular DerivativeForm, that is to say
+   * Return the transpose of a rectangular DerivativeForm,
    * viewed as a two dimensional matrix.
    */
   DerivativeForm<1, spacedim, dim, Number>
   transpose() const;
 
   /**
    * Compute the Frobenius norm of this form, i.e., the expression
-   * $\sqrt{\sum_{ij} |DF_{ij}|^2}$.
+   * $\sqrt{\sum_{ij} |DF_{ij}|^2} =
+   * \sqrt{\sum_{ij} |\frac{\partial F_i}{\partial x_j}|}$.
    */
   typename numbers::NumberTraits<Number>::real_type
   norm() const;
 
   /**
    * Compute the volume element associated with the jacobian of the
-   * transformation F. That is to say if $DF$ is square, it computes
+   * transformation $\mathbf F$. That is to say if $DF$ is square, it computes
    * $\det(DF)$, in case DF is not square returns $\sqrt{\det(DF^T * DF)}$.
    */
   Number
   determinant() const;
 
   /**
    * Assuming that the current object stores the Jacobian of a mapping
-   * $F$, then the current function computes the <i>covariant</i> form
-   * of the derivative, namely $(\nabla F)G^{-1}$, where $G = (\nabla
-   * F)^{T}*(\nabla F)$. If $\nabla F$ is a square matrix (i.e., $F:
+   * $\mathbf F$, then the current function computes the <i>covariant</i> form
+   * of the derivative, namely $(\nabla \mathbf F) {\mathbf G}^{-1}$, where
+   * $\mathbf G = (\nabla \mathbf F)^{T}*(\nabla \mathbf F)$. If $\nabla \mathbf
```

Comment:
Let's get rid of the asterisk in the middle while we're here:
```suggestion
   * $\mathbf G = (\nabla \mathbf F)^{T}(\nabla \mathbf F)$. If $\nabla \mathbf
```

Reply:
Yes, we should remove the asterisk because we are dealing with tensors.